content/en/docs/ops/best-practices/security/index.md

```diff
@@ -7,36 +7,170 @@ owner: istio/wg-security-maintainers
+    name: http
+    protocol: HTTP
+  hosts:
+  - "foo.example.com" # Allow only VirtualServices that are for foo.example.com
+  - "default/bar.example.com" # Allow only VirtualServices in the default namespace that are for bar.example.com
+  - "route-namespace/*" # Allow only VirtualServices in the route-namespace namespace for any host
+{{< /text >}}
+
+### Isolate sensitive services
+
+It may be desired to enforce stricter physical isolation for sensitive services. For example, you may want to run a
+[dedicated gateway instance](/docs/setup/install/istioctl/#configure-gateways) for a sensitive `payments.example.com`, while utilizing a single
+shared gateway instance for less sensitive domains like `blog.example.com` and `store.example.com`.
+
+## Protocol detection
+
+Istio will [automatically determine the protocol](/docs/ops/configuration/traffic-management/protocol-selection/#automatic-protocol-selection) of traffic it sees.
+To avoid accidental or intentional miss detection, which may result in unexpected traffic behavior, it is recommended to [explicitly declare the protocol](/docs/ops/configuration/traffic-management/protocol-selection/#explicit-protocol-selection) where possible.
+
+## CNI
+
+In order to transparently capture all traffic, Istio relies on `iptables` rules configured by the `istio-init` `initContainer`.
+This adds a [requirement](/docs/ops/deployment/requirements/) for the `NET_ADMIN` and `NET_RAW` [capabilities](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-capabilities-for-a-container) to be available to the pod.
+
+To reduce privileges granted to pods, Istio offers a [CNI plugin](/docs/setup/additional-setup/cni/) which removes this requirement.
+
+{{< warning >}}
+The Istio CNI plugin is currently an alpha feature.
+{{< /warning >}}
+
+## Use hardened docker images
+
+Istio's default docker images, including those run by the control plane, gateway, and sidecar proxies, are based on `ubuntu`.
+This provides various tools such as `bash` and `curl`, which trades off convenience for an increase attack surface.
+
+Istio also offers a smaller image based on [distroless images](/docs/ops/configuration/security/harden-docker-images/) that reduces the dependencies in the image.
+
+{{< warning >}}
+Distroless images are currently an alpha feature.
+{{< /warning >}}
+
+## Release and security policy
+
+In order to ensure your cluster has the latest security patches for known vulnerabilities, it is important to stay on the latest patch release of Istio and ensure that you are on a [supported release](/about/supported-releases) that is still receiving security patches.
+
+## Avoid alpha and experimental features
+
+All Istio features and APIs are assigned a [feature status](/about/feature-stages/), defining its stability, deprecation policy, and security policy.
+
+Because alpha and experimental features do not have as strong security guarantees, it is recommended to avoid them whenever possible.
```

Comment:
Perhaps make this stronger.  We do not create security releases for vulnerabilities in experimental features.  We decide on a case by case basis whether to fix vulnerabilities in alpha features (basically, if we think there is broad adoption of an alpha feature, we'll patch vulnerabilities following principles of responsible disclosure)